package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Offers",
-  "version": "1.0.4",
+  "version": "0.1.6",
```

Comment:
Please be sure to view diff of changes before each commit to avoid committing this